client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -0,0 +1,228 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { noop, pick } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormLegend from 'components/forms/form-legend';
+import FormRadio from 'components/forms/form-radio';
+import FormCountrySelect from 'components/forms/form-country-select';
+import FormTextInput from 'components/forms/form-text-input';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import { forDomainRegistrations as countriesListForDomainRegistrations } from 'lib/countries-list';
+
+const countriesList = countriesListForDomainRegistrations();
+
+class RegistrantExtraInfoForm extends React.PureComponent {
+	static propTypes = {
+		isVisible: PropTypes.bool,
+		onSubmit: PropTypes.func,
+		countriesList: PropTypes.object,
+	}
+
+	static defaultProps = {
+		isVisible: false,
+		isProbablyOrganization: false,
+		onSubmit: noop,
+		countriesList: countriesList,
+	}
+
+	constructor( props ) {
+		super( props );
+		this.handleSubmit = this.handleSubmit.bind( this );
+	}
+
+	componentWillMount() {
+		this.state = {
+			registrantType: this.props.isProbablyOrganization
+				? 'organization' : 'individual',
+			countryOfBirth: 'AU',
+			dateOfBirth: '1978-01-01',
+			placeOfBirth: 'dummyCity',
+			postalCodeOfBirth: '12345',
+			registrantVatId: 'XX123456789',
+			sirenSiret: '123456789',
+			trademarkNumber: '123456789',
+		};
+	}
+
+	getRelevantFields( state ) {
+		const { countryOfBirth, registrantType } = state;
+		const bornInFrance = countryOfBirth === 'FR';
+		const birthPlaceFields = [ 'placeOfBirth', 'postalCodeOfBirth' ];
+		const individualFields = [
+			'countryOfBirth',
+			'dateOfBirth',
+			...( bornInFrance ? birthPlaceFields : [] ),
+		];
+		const organizationalFields = [ 'registrantVatId', 'sirenSiret', 'trademarkNumber' ];
+		const conditionalFields = registrantType === 'individual'
+			? individualFields
+			: organizationalFields;
+		return [ 'registrantType', ...conditionalFields ];
+	}
+
+	handleSubmit() {
+		const relevantFields = this.getRelevantFields( this.state );
+		// TODO: Validate first
+		// TODO: Do we need to strip spaces from VAT or SIRE[NT]?
+		const result = pick( this.state, relevantFields );
+		this.props.onSubmit( result );
+	}
+
+	render() {
+		return (
+			<div>
+				<Card className="registrant-extra-info__title-card">
+						{ this.props.translate( '.FR Registration' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **66** times:
`translate( 'Registration' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/tld/fr-extra-fields-dialog)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).